docs/Guide/plugins/Subcommands/subcommand-groups.mdx

```diff
@@ -1,49 +1,27 @@
 ---
-title: Using Subcommand groups
+title: Using Subcommand Groups for Chat Input Commands
 sidebar_position: 2
 ---
 
-Before we start, we assume you already know what subcommand groups are, if not please refer to the
-[Official Discord Docs](https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups)
-for more information.
+The following example will build off the command in the previous section. As a disclaimer, the result on the user side
+will seem redundant when compared to the previous command. Regardless, the purpose is to show how to use Subcommand
+Groups, rather than a useful example.
 
-## Message Subcommand Groups
+Since Subcommand Groups are a little nuanced, please refer to Discord's
+[Subcommands and Subcommand Groups](https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups)
+documentation to see better examples of applying them.
```

Comment:
```suggestion
Subcommand groups are a way to...well...group subcommands by some criteria you want (for instance all subcommands affecting users can go into one `user` group). Discord's
[Subcommands and Subcommand Groups](https://discord.com/developers/docs/interactions/application-commands#subcommands-and-subcommand-groups) explanation might give you some better examples for how you can use them for your users' benefits! Here, we are just showing you how to implement the mappings for them using the Subcommands plugin! 
```

Reply:
I rewrote this when adding the message commands section, let me know if there are other changes needed.